tests/example_listings.py

```diff
@@ -7,31 +12,28 @@
 settings.load_profile("unit")
 
 
+@composite
+def requirements_list(draw, length, answers=False):
+    if answers:
+        elements = booleans() if length is not None else one_of(booleans(), none())
+    else:
+        elements = text(min_size=1, alphabet='abcdefgh', average_size=10)
```

Comment:
Does reducing the alphabet not also reduce the effectiveness of the tests. We want the API to not blow up when bad data is passed in.